audio/rsgain/Portfile

```
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                      1.0
+
+PortGroup                       github 1.0
+PortGroup                       cmake 1.1
+
+github.setup                    complexlogic rsgain 3.5.2 v
+
+categories                      audio
+platforms                       darwin freebsd linux netbsd openbsd
```

Perhaps this might better be:

```suggestion
platforms                       {darwin all}
```